scripts/launch/demo.sh

```diff
@@ -78,9 +78,17 @@ fi
 # and start processing the first slot.
 if [ -z "$system_start" ]
   then
-    system_start=$((`date +%s` + 15))
+    system_start=$((`date +%s` + 20))
```

Comment:
Why is it necessary?

Reply:
@ksaric Reported whilst working on CSL-1951/CSL-1970 that in order for the `wallet-new` executable to work within the old launch script such bump was needed (at least on his machine). This can be plausible considering either a slower machine than average or the fact that `wallet-new` also writes on disk the updated swagger spec, and maybe that's the culprit.

However, if you think this is a roadblock in merging this PR I'm happy to revert, Mr @gromakovsky 😉

Reply:
I don't mind merging this PR with this change (because I feel your pain that you can't merge it for so long 😅), but I'd like to know what takes this time.
Previously there was 40 sec delay and it was really annoying to wait for 40 sec when you launch demo to test that things work. Then we investigated it and came to conclusion that 15 sec should be enough for everybody. Now it's going into wrong direction again and it's better not to let it be 40 sec :)